Assets/Scripts/Models/Trade/Currency.cs

```diff
@@ -8,13 +8,31 @@
 #endregion
 
 using System.Xml;
+using System;
 
 public class Currency
 {
     public string Name;
     public string ShortName;
+    
     public float Balance;
     
+    public void SetBalance (float value) {
```

Comment:
Should this be a property with a getter/setter that calls balanceChanged? I'm not sure myself, so hopefully someone can give another opinion, just in case we have a guideline that prefers one or the other.